src/components/icon/icon.spec.js

```diff
@@ -419,6 +419,29 @@ describe('mdIcon service', function() {
         $scope.$digest();
       });
 
+      describe('and the URL is a data URL', function() {
+        var svgData = '<svg><g><circle r="50" cx="100" cy="100"></circle></g></svg>';
+
+        describe('and the data is base64 encoded', function() {
+          it('should return correct SVG markup', function() {
+            var data = 'data:image/svg+xml;base64,' + btoa(svgData);
```

Comment:
I'm not sure I follow. Are you saying you want a test that passes this pre-encoded data
```
var data = 'data:image/svg+xml;base64,PHN2Zz48Zz48Y2lyY2xlIHI9IjUwIiBjeD0iMTAwIiBjeT0iMTAwIj48L2NpcmNsZT48L2c+PC9zdmc+';
```
rather than encoding on-the-fly (as below)?

```
var data = 'data:image/svg+xml;base64,' + btoa(svgData);
```